### PR TITLE
Create multiple records

### LINKED ISSUE #1: Create multiple records with the vane helper
Currently with `sails.helpers.vane` one can create a single record. It will be helpful to be able to create more than one record with a call to the same helper. This might be useful for seeding the database.

### PR DESCRIPTION
**Pull request by haastrupea**
Fixes #1 💯  Create multiple records with the vane helper

### CODE REVIEW

**Review by DominusKelvin — CHANGES_REQUESTED** [review]
Looking good @haastrupea. Thanks for taking the time to submit this PR. Do you think we can rename `totalRecord` to `count` and I like that it is defaulting to 1. I'm looking at simplifying the code and just default to using .createEach even if the caller want's a single record. What do you think?

**Comment by haastrupea** [comment]
> Looking good @haastrupea. Thanks for taking the time to submit this PR. Do you think we can rename `totalRecord` to `count` and I like that it is defaulting to 1. I'm looking at simplifying the code and just default to using .createEach even if the caller want's a single record. What do you think?

Thank you. I thought about it too, it is actually a great way to simplify the code

**Comment by haastrupea** [comment]
> Looking good @haastrupea. Thanks for taking the time to submit this PR. Do you think we can rename `totalRecord` to `count` and I like that it is defaulting to 1. I'm looking at simplifying the code and just default to using .createEach even if the caller want's a single record. What do you think?

Done

**Comment by DominusKelvin** [comment]
@haastrupea this is getting exciting. I was wondering if we can smartly return an object if the record created was 1 and we can take that cognitive load off the user. What do you think?

Also, can you attach a screenshot of this working on your machine?

**Comment by haastrupea** [comment]
> @haastrupea this is getting exciting. I was wondering if we can smartly return an object if the record created was 1 and we can take that cognitive load off the user. What do you think?
> 
> Also, can you attach a screenshot of this working on your machine?



Well, I thought about it too but you know the rule, always keep it simple. Now that you asked, it is done.
Just so you know for readability I decided to go with the `if` block for the logic instead of a `ternary` operator. if that is okay.

File: helper/vane.js [hunk 1]
```diff
@@ -27,7 +31,17 @@ module.exports = {
...
+    const model = sails.models[model]
+    if(totalRecord <= 1){
+     result =  await model.create(initialValues).fetch();
+    }else {
+      const initialValuesArray = [];
+      for (let i = 0; i < totalRecord; i++) {
+        initialValuesArray.push(initialValues)
+      }
+      result = await model.createEach(initialValuesArray).fetch();
+    }
```
Comment (DominusKelvin) [inline helper/vane.js]: `return await model.create(initialValues).fetch()`

**Comment by DominusKelvin** [comment]
Thanks @haastrupea

**Review by DominusKelvin — APPROVED** [review]
Great job on this one

**Pull request merged**